dll/win32/shell32/lang/en-US.rc

```diff
@@ -911,6 +911,10 @@ BEGIN
     FCIDM_SHVIEW_NEWFOLDER "New &Folder"
     FCIDM_SHVIEW_NEWLINK "New &Link"
     IDS_FOLDER_OPTIONS "Folder Options"
+    IDS_TASKBAR_OPTIONS "Taskbar and Start Menu"
+    IDS_TASKBAR_OPTIONS_INFOTIP "Customize the Start Menu and the taskbar, such as the types of items to be displayed and how they should appear."
+    IDS_TASKBAR_OPTIONS "Taskbar and Start Menu"
+    IDS_TASKBAR_OPTIONS_INFOTIP "Customize the Start Menu and the taskbar, such as the types of items to be displayed and how they should appear."
```

Comment:
```suggestion
```

Reply:
Ah yes, make English string. Then begrudgingly add all the other pointless translations and make a mistake.